client/views/admin/settings/groups/voip/VoipExtensionsPage.tsx

```diff
@@ -86,7 +86,11 @@ const VoipExtensionsPage: FC = () => {
 						{queues?.length > 2 && `+${(queues.length - 2).toString()}`}
 					</Box>
 				</Table.Cell>
-				{username ? <RemoveAgentButton username={username} reload={reload} /> : <AssignAgentButton extension={extension} reload={reload} />}
+				{username ? (
+					<RemoveAgentButton username={username} reload={reload} extension={extension} />
```

Comment:
Since `handleExtensionStateChange` needs a bunch of data that exists on this file, I think it would be better to declare `handleExtensionStateChange` in this component and pass the function as props, instead of passing the username and requesting the user id inside `removeAgentButton`